hack/flux/update-flux.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+IFS=$'\n\t'
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+readonly REPO_ROOT
+LATEST_FLUX_VERSION="$(gh api -X GET "repos/fluxcd/flux2/releases" --jq '.[0].tag_name|sub("^v"; "")')"
+readonly LATEST_FLUX_VERSION
+CURRENT_FLUX_VERSION=$(find "${REPO_ROOT}/services/kommander-flux" -maxdepth 1 -regextype sed -regex '.*/[0-9]\+.[0-9]\+.[0-9]\+' -printf "%f\n" | sort -V | head -1)
+readonly CURRENT_FLUX_VERSION
+
+function update_flux() {
+    readonly BRANCH_NAME="flux-update/${LATEST_FLUX_VERSION}"
+    git checkout -b "${BRANCH_NAME}"
+
+    asdf install flux2 "${LATEST_FLUX_VERSION}"
+    asdf local flux2 "${LATEST_FLUX_VERSION}"
+
+    mkdir -p "$REPO_ROOT/services/kommander-flux/$LATEST_FLUX_VERSION"
+    pushd "$REPO_ROOT/services/kommander-flux/$LATEST_FLUX_VERSION"
+    ls ..
+    cp -a ../"$CURRENT_FLUX_VERSION"/* .
```

Comment:
was it intentional to NOT remove the current flux version directory?

Reply:
Oops, not exactly. I believe it doesn't hurt having it there, though.

Reply:
i just tried this locally and, (rather untentionally) created #48 but it has both the versions of flux.

Reply:
if we keep multiple versions of flux, the diff is harder to review (like in #48) also not sure if https://github.com/mesosphere/kommander/blob/main/Makefile#L30 would break anytime in future (sort may behave weird if semantic versioning is not strictly followed), i ll create a follow up PR to remove the old directory and that should keep things simple.

Reply:
Thanks @takirala!!

Reply:
https://github.com/mesosphere/kommander-applications/pull/49/files